RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@
 * [*] Disables the ability to open the editor for Post Pages [#14523]
 * [*] Block Editor: "Set as featured" button within image block settings. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3116]
 * [***] Block Editor: Audio block now available on WP.com sites on the free plan. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3523]
+* [*] Block Editor: Improve unsupported block message for reusable block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3621]
```

Comment:
Just FYI, note that this won't made it to the release notes, since the release notes for 17.6 had already been send to Editorial at the beginning of last week, and sent to our translation vendor since then, so it's too late to include new entries at that point in the release 🤷